ui/app/first-time/init-menu.js

```diff
@@ -128,6 +129,17 @@ InitializeMenuScreen.prototype.renderMenu = function (state) {
         }, 'I already have a DEN that I would like to import'),
       ]),
 
+      state.forgottenPassword ? h('.flex-row.flex-center.flex-grow', [
+        h('p.pointer', {
+          onClick: this.backToUnlockView.bind(this),
+          style: {
+            fontSize: '0.8em',
+            color: 'rgb(247, 134, 28)',
+            textDecoration: 'underline',
+          },
+        }, 'I remember my password!'),
```

Comment:
I don't like exclaimation marks in UI.

Could you just make this `Log in`?

Also, put it before the import den item.